lib/AArch64Base.ml

```diff
@@ -1472,10 +1472,8 @@ let do_pp_instruction m =
       pp_mem_ext "LDRSW" V64 r1 r2 idx
   | I_LDRS ((v,bh),r1,r2,idx) ->
      pp_mem_ext (ldrs_memo bh) v r1 r2 idx
-  | I_LDUR (_,r1,r2,None) ->
-      sprintf "LDUR %s, [%s]" (pp_reg r1) (pp_reg r2)
-  | I_LDUR (_,r1,r2,Some(k)) ->
-      sprintf "LDUR %s, [%s, %s]" (pp_reg r1) (pp_reg r2) (m.pp_k k)
+  | I_LDUR (_,r1,r2,k) ->
+      sprintf "LDUR %s, [%s%s]" (pp_reg r1) (pp_reg r2) (pp_kr false false (K k))
```

Comment:
It's this space that makes the hash change? I think it's fine unless @maranget has runs with LDUR and the change of hash causes problems for his database.